src/CleanArchitecture.Web/CleanArchitecture.Web.csproj

```diff
@@ -1,54 +1,23 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <Sdk Name="Microsoft.Build.CentralPackageVersions" Version="2.0.46" />
+  
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <PreserveCompilationContext>true</PreserveCompilationContext>
-    <AssemblyName>CleanArchitecture.Web</AssemblyName>
     <OutputType>Exe</OutputType>
-    <PackageId>CleanArchitecture.Web</PackageId>
+    <WebProjectMode>true</WebProjectMode>
```

Comment:
What's this?

Reply:
Custom flag. Common setup for the web can be shared and by enabling flag we don't need to copy-paste configuration in each project. This setup can be found here: https://github.com/ardalis/CleanArchitecture/blob/f97e3367e8da6a330889dbeea0010ad5829060ca/Directory.Build.props#L31-L44

This same approach for tests:
https://github.com/ardalis/CleanArchitecture/blob/f97e3367e8da6a330889dbeea0010ad5829060ca/tests/CleanArchitecture.Tests/CleanArchitecture.UnitTests.csproj#L7
https://github.com/ardalis/CleanArchitecture/blob/f97e3367e8da6a330889dbeea0010ad5829060ca/tests/CleanArchitecture.IntegrationTests/CleanArchitecture.IntegrationTests.csproj#L6
https://github.com/ardalis/CleanArchitecture/blob/f97e3367e8da6a330889dbeea0010ad5829060ca/tests/CleanArchitecture.FunctionalTests/CleanArchitecture.FunctionalTests.csproj#L6

setup:
https://github.com/ardalis/CleanArchitecture/blob/f97e3367e8da6a330889dbeea0010ad5829060ca/Packages.props#L22-L32
and
https://github.com/ardalis/CleanArchitecture/blob/f97e3367e8da6a330889dbeea0010ad5829060ca/Directory.Build.props#L46-L48

Reply:
This approach gives predictable results because scripts, custom setups are defined inside a shared solution so the configuration of the additional projects is dynamic and simple in most cases - we know what we want to from tests, web, libraries.